src/vs/workbench/parts/files/electron-browser/explorerService.ts

```diff
@@ -108,8 +108,13 @@ export class ExplorerService implements IExplorerService {
 		return this.model.findClosest(resource);
 	}
 
-	setEditable(stat: ExplorerItem, data: IEditableData): void {
-		this.editableStats.set(stat, data);
```

Comment:
There was an issue with @isidorn's model here. This function is often called with `setEditable(stat, null)` to clear a stat's editable data, so we would set `stat => null` in the map, which means the `isEditable()` call further down would return true... even when we cleared editable data for a stat.